src/js/controllers/oppController.js

```diff
@@ -62,149 +62,111 @@ export const oppScreen = async function(state){
-                    <li class="referral">
-                        <h3 class='opp-category'>SUBJECT CATEGORY</h3>
-                        <div>
-                            <h2>Job Title</h2>
-                            <h3>Company Name</h3>
-                            <h3 class="opp-alumni">from Alumni Name</h3>
-                        </div>
-                    </li>
-                    <li class="referral">
-                        <h3 class='opp-category'>SUBJECT CATEGORY</h3>
-                        <div>
-                            <h2>Job Title</h2>
-                            <h3>Company Name</h3>
-                            <h3 class="opp-alumni">from Alumni Name</h3>
-                        </div>
-                    </li>
+
                 </ul>
             </div>
         </div>   
     `;
 
-    elements.container.insertAdjacentHTML('beforeend',oppSetUp);
-    controlOpp(state);
-    console.log("bill function started" + state.user.name);
-
-    document.querySelector('.referral-box').addEventListener('click', e => {
-        const btn = e.target.closest('.referral');
-        //const oppQuery = `?id=${btn.id}`;
-        if(btn) {
-            // clear right screen
-            clearScreen();
-            // render temp 2
-            oppView.renderDetail(state.opp.opps[btn.id]);
-
-            document.querySelector('.back-top').addEventListener('click', e => {
-                const btn2 = e.target.closest('.back-top');
-
-                if(btn) {
-                    // clear right screen
-                    clearScreen();
-                    // render temp 2
-                    billFunction(state,oppSetUp);
+    const billFunction = (state) => {
+        elements.container.insertAdjacentHTML('beforeend',oppSetUp);
+        controlOpp(state);
+
+        document.querySelector('.referral-box').addEventListener('click', e => {
+            const btn = e.target.closest('.referral');
+            if(btn) {
+                // clear right screen
+                clearScreen();
+                // render temp 2
+                oppView.renderDetail(state.opp.opps[btn.id]);
+
+                document.querySelector('.back-top').addEventListener('click', e => {
+                    const btn2 = e.target.closest('.back-top');
+
+                    if(btn) {
+                        // clear right screen
+                        clearScreen();
+                        // render temp 2
+                        billFunction(state,oppSetUp);
+                    }
+                });
+
+                //console.log("attention: " + state.user.uid);
+                //console.log("attention: " + state.opp.opps[btn.id].data().registered[1].uid);
+                let flag = false;
+                for (let i=0; i<state.opp.opps[btn.id].data().registered.length; i++){
+                    if (state.opp.opps[btn.id].data().registered[i].uid == state.user.uid){
+                        flag = true;
+                    }
```

Comment:
Can we get some documentation on what's going on here?

Reply:
I understand that it's checking all people registered to see if I am registered but documentation is needed.

Reply:
This problem has been solved in our latest version.